.github/actions/update-viablestrict/action.yml

```diff
@@ -0,0 +1,96 @@
+name: Update viable/strict to the latest green commit from a PyTorch repo
+
+description: |
+  The latest green commit is the newest commit that have passed all the required
+  test in trunk and can be consider stable. This GHA only works for repos that
+  have been onboard to PyTorch Dev Infra.
+
+inputs:
+  repository:
+    description: The repository name, i.e. pytorch/pytorch
+    required: true
+    type: string
+  stable-branch:
+    description: The name of the stable branch to push to
+    required: false
```

Comment:
Why isn't it required?

Reply:
Right, I could make this default to `viable/strict` and make it an optional parameter.

Reply:
Ah wait, this is already not required.  I'm having a brainfart lol, this is not required because there is already a default value, ET could choose to name the stable branch viable/strict like PT, or something else.